dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/DdSdkConfiguration.kt

```diff
@@ -21,5 +21,5 @@ data class DdSdkConfiguration(
     val applicationId: String? = null,
     val nativeCrashReportEnabled: Boolean? = null,
     val sampleRate: Double? = null,
-    val additionalConfig: Map<String, Any?>? = null
+    val additionalConfig: Map<String, Any>? = null
```

Comment:
This signature should not be changed by hand, this file is automatically generated.

Reply:
thanks, didn't know it is auto-generated.